renderer.go

```diff
@@ -36,40 +36,12 @@ type renderer interface {
 	// Hide the cursor.
 	hideCursor()
 
-	// enableMouseCellMotion enables mouse click, release, wheel and motion
-	// events if a mouse button is pressed (i.e., drag events).
-	enableMouseCellMotion()
```

Comment:
one key difference in all these methods is that they were locking the renderer before writing... not sure if we need that, but i thought it was worth mentioning anyway

Reply:
It won't matter because the renderer eventually writes to the program output, _not_ the renderer buffer